third_party/blink/renderer/core/core_initializer.cc

```diff
@@ -92,6 +92,7 @@ void CoreInitializer::Initialize() {
   // Initialize must be called once by singleton ModulesInitializer.
   DCHECK(!instance_);
   instance_ = this;
+  // CHECK(false) << "COre init";
```

Comment:
remove?

Reply:
Thanks for catching, this is cruft that got added accidentally to the commit